rephrase list of support resources and add IRC

v4hn opened the pull request:
I'd like to have the IRC channel mentioned on the web page
and I figured the other items could use an overhaul.

davetcoleman commented on support/index.markdown:
```diff
@@ -14,9 +14,9 @@ title: Support
-  * Join [the MoveIt! users mailing list](https://groups.google.com/forum/#!forum/moveit-users) (moveit-users _at_ googlegroups _dot_ com) for receiving announcements from [here](https://groups.google.com/forum/#!forum/moveit-users/join). In the [list page](https://groups.google.com/forum/#!forum/moveit-users) you can also find previous good discussions.
-    * (September 2016) We are transitioning to no longer using the mailing list for basic user questions but only for announcements and high level milestone questions.
+  * Ask questions at [answers.ros.org](http://answers.ros.org/), the official question forum of ROS and MoveIt!, if you haven't found similar questions.
+  * Join the IRC channel #moveit in the [FreeNode](http://freenode.net) network for casual discussion.
```
The mailing list is still more important/modern/utilized than the IRC channel, it should stay as second on the list to imply that, not third.

v4hn commented:
You're right. I just thought that the mailing list is
not intended as a place to "Find Troubleshooting Information" anymore.
Because this is the section header, I moved it to the bottom of the list.

I changed it again.

davetcoleman commented:
Thanks!
